Sources/PointFree/Account/Account.swift

```diff
@@ -480,10 +479,11 @@ private let subscriptionPaymentInfoView = View<Stripe.Subscription> { subscripti
   ]
 }
 
-private func totalAmount(for subscription: Stripe.Subscription) -> String? {
+private func totalAmount(for subscription: Stripe.Subscription) -> String {
   let totalCents = subscription.plan.amount.rawValue * subscription.quantity
-  let totalDollars = Double(totalCents) / 100
-  return currencyFormatter.string(from: NSNumber(value: totalDollars))
+  let totalDollars = NSNumber(value: Double(totalCents) / 100)
+  return currencyFormatter.string(from: totalDollars)
+    ?? NumberFormatter.localizedString(from: totalDollars, number: .currency)
```

Comment:
`NumberFormatter.string(from:)` returns an optional, but `localizedString(from:)` does not 😕